modules/dr-restoring-cluster-state.adoc

```diff
@@ -169,7 +174,7 @@ $ sudo ip address del <reported_vip> dev <reported_vip_device>
 
 . Access the recovery control plane host.
 
-. If the `keepalived` daemon is in use, verify that the recovery control plane node owns the VIP:
+. If the `keepalived` daemon is in use, verify that the recovery control plane node owns the VIP. Otherwise, repeat step 4.xi.
```

Comment:
what's "step 4.xi"? I have not find this step in context.

Reply:
Hi @geliu2016 - I provided this explanation in the first comment in this PR: 
>In the preview for this PR, the substeps use a.,b.,c., etc. However, when this documentation is published on docs.redhat.com, the substeps use i., ii., iii., etc.

>In step 6 of this procedure, step 4.xi is mentioned. In the preview, the equivalent of that step is 4.k.

Reply:
Sure, 
/lgtm